code/game/machinery/stasis.dm

```diff
@@ -144,9 +144,9 @@
 		return
 	var/mob/living/L_occupant = occupant
 	if(stasis_running())
-		if(!IS_IN_STASIS(L_occupant))
+		if(!HAS_TRAIT(L_occupant, TRAIT_STASIS))
```

Comment:
make it `HAS_TRAIT_FROM` `STASIS_MACHINE_TRAIT`, in case someone got stasis from other sources

Reply:
good catch